mne/epochs.py

```diff
@@ -2064,6 +2064,289 @@ def _drop_log_stats(drop_log, ignore=('IGNORED',)):
     return perc
 
 
+def make_metadata(events, event_id, tmin, tmax, sfreq,
+                  time_locked_events=None, keep_first=None, keep_last=None):
```

Comment:
```suggestion
                  epochs_event_id=None, keep_first=None, keep_last=None):
```
I find it more explicit that it's the event_id that will be used to create epochs.
Also it could be a dict or a list. If it's dict you can just call list on it.

Reply:
@drammock suggested to change this too, and proposed to use the terms "row_events" (instead of time_locked_events) and "column_events" (instead of event_id).

Not sure which proposal is better, but you two independently suggesting a name change makes it clear **something** should be changed :)

Reply:
> Also it could be a dict or a list. If it's dict you can just call list on it.

Not sure if we need to support the `dict` case, as we're returning a new `event_id` dict anyway, so there's no need for users to create it manually